stylesheets/colorscheme.css

```diff
@@ -0,0 +1,12 @@
+:root {
+    --color-main-back: #F5BD55;
+    --color-title-back: #1E8FB3;
+    --color-link-hover:#1E8FB3;
+    --color-footer-back:#1E8FB3;
+    --color-title-font: #F4E8C1;
+    --color-link:#F4E8C1;
+    --color-footer-link:#F4E8C1;
+    --color-picback-hover:#B15A51; 
+    --color-footer-link-hover:#B15A51;
```

Comment:
I recommend using a different color here than the background, to avoid disappearing link illusion.